src/relay/mod.rs

```diff
@@ -7,36 +7,49 @@ use tokio::net::TcpListener;
 use tokio::net::ToSocketAddrs;
 
 mod api;
+mod federation;
 mod switch;
-
 use api::RelayHook;
+pub use federation::Federation;
```

Comment:
no need for `pub`

Reply:
I need it pub to initialize it in rmb-relay.rs